modules/core/src/main/scala/fs2/kafka/producer/MkProducer.scala

```diff
@@ -17,8 +17,8 @@ import org.apache.kafka.common.serialization.ByteArraySerializer
   * the fs2-kafka `KafkaProducer`. This is needed in order to instantiate
   * [[fs2.kafka.KafkaProducer]] and [[fs2.kafka.TransactionalKafkaProducer]].<br><br>
   *
-  * By default, the instance provided by [[MkProducer.mkProducerForSync]] will be used. However this
-  * behaviour can be overridden, e.g. for testing purposes, by placing an alternative implicit
+  * By default, the instance provided by [[MkProducer.mkProducerForSync]] will be used. However,
```

Comment:
I couldn't help myself here. Please let me know if you prefer to keep this change in a different PR.

Reply:
Hahaha, LGTM 👍🏽